packages/ember-routing/lib/system/router.js

```diff
@@ -411,8 +411,8 @@ const EmberRouter = EmberObject.extend(Evented, {
     return this.transitionTo(...arguments).method('replace');
   },
 
-  generate() {
-    let url = this._routerMicrolib.generate(...arguments);
+  generate(...args) {
+    let url = this._routerMicrolib.generate(...args);
```

Comment:
I added this while I was fiddling with the params to ensure they were passed through to `_routerMicrolib.generate` in the expected format. I realized that this method handles extracting the params correctly already, so this is just a remnant of that. Will revert for sure!